src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -399,15 +401,15 @@ public void zadd(final byte[] key, final double score, final byte[] member) {
     sendCommand(ZADD, key, toByteArray(score), member);
   }
 
-  public void zaddBinary(final byte[] key, final Map<byte[], Double> scoreMembers) {
+  public void zadd(final byte[] key, final double score, final byte[] member,
+      final ZAddParams params) {
+    sendCommand(ZADD, params.getByteParams(key, toByteArray(score), member));
+  }
 
+  public void zaddBinary(final byte[] key, final Map<byte[], Double> scoreMembers) {
```

Comment:
@marcosnils 
Great finding! Renamed zaddBinary to zadd.
Maybe it was just a workaround if Java compiler or JVM could not distinguish methods if only param's Generic is different.
For now, all tests passed.